CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Multiple open instances of the extension could potentially clobber writes to
   `~/.ssh/config`. Updates to this file are now atomic.
 - Add support for `anysphere.remote-ssh` Remote SSH extension.
+- Use `--header-command` properly when starting a workspace.
```

Comment:
I think you meant to put this in the unreleased section!

Reply:
it's dark and scary in here, I got lost 😭